docs/developer/eventing/sinks/creating-event-sinks/writing-event-sinks-with-golang/README.md

```diff
@@ -0,0 +1,330 @@
+---
+title: "Writing an event sink using Golang"
+weight: 10
+type: "docs"
+showlandingtoc: "false"
+---
+
+# Writing an event sink using Golang
+
+This tutorial provides instructions to build an event sink in Golang and implement it as a ksvc.
```

Comment:
please write "Knative Service" rather than "ksvc" to avoid confusion

Reply:
Fixed: https://github.com/knative/docs/pull/4208/commits/b8225598113892d3f55f56b0d5d006485761e575 

Thank you!